apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -1318,6 +1318,28 @@ async fn handle_json_array_body(
 
     debug!("parsed json array for {}", topic_name);
 
+    // Log payload if enabled (compact JSON on one line)
+    if std::env::var("MOOSE_LOG_PAYLOADS")
```

Comment:
Could we preload it?

This will get re-executed for every requests which we get a lot of